themes/vocabulary_theme/templates/blocks/hero.html

```diff
@@ -0,0 +1,23 @@
+<section>
```

Comment:
I think it's a good idea give it a unique name to this module... maybe something like `homepage-hero` or so